internal/operators/ocs/validations.go

```diff
@@ -232,7 +232,7 @@ func getValidDiskCount(disks []*models.Disk) int {
 	var countDisks int
 
 	for _, disk := range disks {
-		if disk.SizeBytes >= conversions.GbToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
+		if disk.SizeBytes >= conversions.GibToBytes(MinDiskSize) && (disk.DriveType == ssdDrive || disk.DriveType == hddDrive) {
```

Comment:
The MinDiskSize has not been changed to GiB in the constant definition. We are still using GB. (See line 220 as well)

Reply:
+1

Reply:
Done